src/Sarif.Multitool.FunctionalTests/TestData/UriBaseIdRequiresRelativeUri/AbsoluteUriInRunGraphNode_Expected.sarif

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "Sarif.Multitool",
+        "fullName": "Sarif.Multitool 2.0.0.0",
+        "version": "2.0.0.0",
+        "semanticVersion": "2.0.0",
+        "sarifLoggerVersion": "2.0.0.0",
+        "language": "en-US",
+        "properties": {
+          "Comments": "Command line tool to perform transformations of input files to SARIF.",
+          "CompanyName": "Microsoft",
+          "ProductName": "Microsoft SARIF SDK"
+        }
+      },
+      "results": [
+        {
+          "ruleId": "SARIF1014",
+          "level": "error",
+          "message": {
+            "arguments": [
+              "/runs/0/graphs/0/nodes/0/location/physicalLocation/fileLocation/uri",
```

Comment:
>runs/0/graphs/0/nodes/0/location/physicalLocation/fileLocation/uri" [](start = 16, length = 67)

this is a logical location presumably! right? :) so why aren't you specifying the fullyQualifiedName or the logical locations table. heh. would be useful test data for some of our UX explorations.

Reply:
Not sure what you're asking. This `_Expected` file shows that the validator detected a problem in the `uri` property of the `fileLocation` property of the `physicalLocation` property of the `location` property of the 0th element of a graph's `nodes` array. There's no logical location involved. `nodes[0].location` could certainly include a logical location, but that's not relevant to the validation rule I'm testing.

---
In reply to: [210452957](https://github.com/Microsoft/sarif-sdk/pull/1018#discussion_r210452957) [](ancestors = 210452957)

Reply:
Oh, _now_ I see what you're asking! You're asking why my validator tool doesn't produce a logical location corresponding to the JPointer! That's a really nice idea.

---
In reply to: [210453784](https://github.com/Microsoft/sarif-sdk/pull/1018#discussion_r210453784) [](ancestors = 210453784,210452957)